src/main/scala/ch/openolitor/stammdaten/reporting/AuslieferungKorbUebersichtReportService.scala

```diff
@@ -68,7 +68,9 @@ trait AuslieferungKorbUebersichtReportService extends AsyncConnectionPoolContext
 
         val hauptAboFarbCodes = (auslieferungReport.entries map { obj: AuslieferungReportEntry => (obj.korb.abotyp.name -> obj.korb.abotyp.farbCode) }).toMap
         val zusatzAboFarbCodes = (auslieferungReport.entries flatMap { obj: AuslieferungReportEntry => (obj.korb.zusatzAbos map { za => (za.abotyp.name -> za.abotyp.farbCode) }) }).toMap
-        val depotTourFarbCodes = (auslieferungReport.entries flatMap { obj: AuslieferungReportEntry => (obj.depot.map(d => (d.name -> d.farbCode.getOrElse("")))) }).toMap
+        val depotFarbCodes = (auslieferungReport.entries flatMap { obj: AuslieferungReportEntry => (obj.depot.map(d => (d.name -> d.farbCode.getOrElse("")))) }).toMap
+        val tourFarbCodes = (auslieferungReport.entries flatMap { obj: AuslieferungReportEntry => (obj.tour.map(d => (d.name -> ""))) }).toMap
```

Comment:
maybe something for the next commit. Depending on how many entries we have we should collect all things in one go. We could introduce a case class `FarbCodes` and introduce a function which returns it

Reply:
yes, that's a good idea. I just made it this way to solve an exception that happens when filtering tours. Tour, does not have farbcode. Another option is to add the farbcode to Tour... I do not know which ones were the requirements